valhalla/baldr/graphtileheader.h

```diff
@@ -165,21 +165,16 @@ class GraphTileHeader {
    * @return Returns the base lat,lon of the tile (degrees).
    */
   midgard::PointLL base_ll() const {
-    GraphId id(graphid_);
-
-    if (id.level() == TileHierarchy::GetTransitLevel().level) {
-      return TileHierarchy::GetTransitLevel().tiles.Base(id.tileid());
-    }
-    return TileHierarchy::levels()[id.level()].tiles.Base(id.tileid());
+    return {base_ll_[0], base_ll_[1]};
   }
 
   /**
    * Sets the base (SW corner) lat,lon of the tile.
    * @param ll  Base lat,lon of the tile.
    */
   void set_base_ll(const midgard::PointLL& ll) {
-    base_ll_.first = ll.lng();
-    base_ll_.second = ll.lat();
+    base_ll_[0] = ll.lng();
```

Comment:
changed it to a std::array from a std::pair. apparently the latter is not generally trivially copyable, depending on the compiler and surprisingly on gcc 15 it's not. initially I had somewhere a memcpy and GraphTileHeader wasn't trivially copyable. now it is.